common/src/main/java/com/wynntils/gui/render/MapRenderer.java

```diff
@@ -434,4 +434,20 @@ public static float getRenderZ(Poi poi, float mapCenterZ, float centerZ, float c
         double distanceZ = poi.getLocation().getZ() - mapCenterZ;
         return (float) (centerZ + distanceZ * currentZoom);
     }
+
+    public static List<MapTexture> getMapTextures(
+            int mapCenterX, int mapCenterZ, float renderWidth, float renderHeight, float scale) {
+        float renderDistanceX = renderWidth / 2f;
+        // scale is the conversion betweeen in-game ("map") coordinates and render coordinates
+        int mapDistanceX = (int) (renderDistanceX * scale);
```

Comment:
mapDistanceX is half render width?

Reply:
i am blind.

Reply:
I think the map rendering code is really tricky. There is no clear notion if coordinates are relative to the Minecraft world, or the screen. All types are the same, int, double or floats. Often, but not always, names indicate coordinate system, but there is no clear and consequent naming standard.